compiler-rt/lib/radsan/tests/radsan_test_interceptors.cpp

```diff
@@ -236,6 +240,72 @@ TEST(TestRadsanInterceptors, readDiesWhenRealtime) {
   std::remove(temporary_file_path());
 }
 
+TEST(TestRadsanInterceptors, writeDiesWhenRealtime) {
+  auto fd = open(temporary_file_path(), O_WRONLY);
+  EXPECT_THAT(fd, Ne(-1));
```

Comment:
This file descriptor is never valid for me, which means that the read and writes are not valid later on. This leads to the test segfaulting on my ubuntu docker image (and I'm surprised it passes on mac).

I'm not sure if they somehow sandbox their unit tests? Like they don't allow you to read/write bytes out? We may consider moving this to a lit test if we keep running into the problem